hack/test-integration.env

```diff
@@ -0,0 +1,6 @@
+# SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+export GOMEGA_DEFAULT_EVENTUALLY_TIMEOUT=5s
+export GOMEGA_DEFAULT_EVENTUALLY_POLLING_INTERVAL=200ms
```

Comment:
was there a special reason to overwrite the defaults?

Reply:
Yes, the default timeout of 1 sec was too short. I did some additional investigation and benchmarking. I will provide the results in a another comment.

Reply:
but is it only failing on the CI system? otherwise I'd suggest to increase the respective timeout in code on where it usually fails or for all in the test suite, so that when you run the debugger the correct values are set. Otherwise you have to remember so set the same env in the debugger. WDYT?